roles/bootnode-nethermind/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Download spec.json and bootnodes.txt
+  get_url: url={{ item }} dest={{ home }}/ mode=0644
+  with_items:
+    - "https://raw.githubusercontent.com/{{ MAIN_REPO_FETCH }}/poa-chain-spec/{{ GENESIS_BRANCH }}/spec.json"
+    - "https://raw.githubusercontent.com/{{ MAIN_REPO_FETCH }}/poa-chain-spec/{{ GENESIS_BRANCH }}/bootnodes.txt"
+
+- name: Copy original chainspec before update
+  copy:
+    remote_src: yes
+    src: "{{ home }}/spec.json"
+    dest: "{{ home }}/spec-nethermind.json"
+    owner: "{{ username }}"
+    group: "{{ username }}"
+
+- name: Update chainspec for nethermind
```

Comment:
When running multiple times it would likely add the same lines over and over again which stands against the idea of idempotency.

Reply:
Thanks for noting. Turns out, this doesn't have any effect because on the previous step `spec-nethermind.json` is copied from original `spec.json`. Same thing happens when unarchiving nethermind zip file and when creating config file.

It might be good to change this behavior, though, so that playbook is "green" after re-run. But I'm not sure what is the best way to do this, because if I use `creates` option (so that task is skipped if file already exists), it won't be updated after original spec.json is updated. Do you have any thought how to do this properly? Or just leave it as it is?

Reply:
Well. Unfortunately, as https://github.com/poanetwork/deployment-playbooks/issues/147 mentions (as well as Google search) - the only way to workaround this is to create another task before the `lineinfile` which will test, that the line is already exists. I'm not sure if this is something we need to do here given the fact that file is being refreshed each time.

Reply:
Thank. Then I think we should leave it as it is now.